apps/frontend/src/pages/auth/libs/components/sign-in-form/sign-in-form.tsx

```diff
@@ -39,7 +41,7 @@ const SignInForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
 					errors={errors}
 					label="Email"
 					name="email"
-					placeholder="name@example.com"
+					placeholder="be@balance.com"
```

Comment:
Return this change back

Reply:
But, in the requirements document we should change this placeholder to "be@balance.com"
![image](https://github.com/user-attachments/assets/95810d7c-2b44-492e-a5c2-cda1b733fe94)

Reply:
Use the previous placeholder, we'll discuss this in our daily